src/components/buttons/LinkButton.js

```diff
@@ -61,6 +37,7 @@ const LinkButton = ({
   ...props
 }) => (
   <StyledLinkButton
+    role="button"
     {...props}
```

Comment:
I assume this "role" is a custom prop we use to set the component display in the DOM  like we did with P.js?

Reply:
accessibility https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role